### ISSUE TITLE
Pumped storage needs a distinct colour

### ISSUE THREAD

**Issue by bje-** [opennem/opennem-fe#168]
Just as batteries use two colours (one for charging and one for discharging), so should pumped storage hydro. Lumping the pumped storage plants in with conventional hydro makes it impossible to see when the pumped storage is operated.

**Comment by dylanjmcconnell** [comment]
I think this is not a bad idea - and something we have discussed to before. The challenge is what to do with Tumut 3 - which does have pumps, but is also predominantly a conventional hydro plant. You would nee to make assumptions around round trip efficiency and some logic on how you might partition "stored" water and "harvested" water - (both of which I have balked at before). 

That said - separating our Wivenhoe and Shoalhaven would be relatively straight forward. 

I would also add it could be an opportunity to add a further category here - 'run of river' hydro. This is largely in Tasmania and northern Queensland (and is essentially a form of variable generation). Distinguishing between "controllable" or "dispatchable" (or what ever word is right) hydro and run of river hydro could also be done. Would add more complexity - but in some ways is not too different to (say) the different forms of gas generation.

**Comment by bje-** [comment]
Yes, that's right: [https://en.wikipedia.org/wiki/Shoalhaven_Scheme](url)
